### DISCUSSION TITLE
Newbie question/need help (trying to get started in headless mode)

### DISCUSSION THREAD

**Discussion by binkomonster** [NVIDIA-AI-IOT/jetbot#387] [General]
Hi.
complete newbie here, trying to get started with the jetson nano, and getting a jetbot pulled together.
Help i need? 
I need to connect my nano to my computer as I want to do setup in headless mode.
However, instructions state "Jumper the J48 Power Select Header pins."
Im *assuming* that this means put a jumper (one of those black 2.54 mm jumper caps) onto the power select header pins right?

**Comment by culebracut** [comment]
if you run headless it uses less power so maybe it can be powered from usb power source.  Not sure if the wifi needs the jumpered power source.

**Comment by santaimpersonator** [comment]
1. Add jumper
2. Plug in micro-USB connection (to computer)
3. Plug in barrel jack power (power supply)

![20210630_212005](https://user-images.githubusercontent.com/36016723/124060022-57e27b80-d9e9-11eb-8ba0-837ece353b87.jpg)
